### ISSUE TITLE
Time agent issues, jecties and typos.

### ISSUE THREAD

**Issue by Eneocho** [d3athrow/vgstation13#31779]
![image](https://user-images.githubusercontent.com/67024428/147930609-1fe24169-5622-4968-ae85-6c2a49a08bee.png)
See Objective N°2.
Instead of "an insulated gloves" it should be "a pair of insulated gloves".
Also the twin has a typo. "Remove John Beckett, the from the timeline with your timeline eraser." 
The jectie should be (in my opinion): "1) Steal your doppelganger's jump charge. 2) Erase him with the timeline eraser. 3) Escape through the time rift."

**Comment by pedr69** [comment]
as the time agent in question, i took a photo of all the shit it wanted me to photo, killed my twin, then the cargo tech, and proceeded to roam maint for 12 minutes because the rift was in the fucking solars behind a plant
then i got deleted by the admin after sending a fax, so that's why it might be broken?
objective 2 was broken before getting deleted though

i think you should spawn in at the rift, or add it to the pinpointer targets(the pinpointer is near useless anyway, can only really track the disk)
also give john a flashlight/nightvision

**Comment by Eneocho** [comment]
If you use the pinpointer in hand it should open a selector to track different things, from crew to (you)r target to the items.

**Comment by pedr69** [comment]
to track crew you need the DNA string(good fuckin luck)
the item track mode could only track the jump charge(it got spaced while i was fumbling in the dark maint)
the disk pinpointer works good
the coordinate mode needs you to input coordinates, and that seems pretty useless

**Comment by Eneocho** [comment]
Yeah, advanced pinpointers need a remake.

**Comment by SonixApache** [comment]
on this note

I completed my chronograph jectie and then had to extract, but I didn't get a message about where the time rift should be. Or if I did, I never saw it. Dunno if it was stored in my notes either. Asked the crew and they didn't see anything time-related either.

Coordinate mode on the pinpointer should be replaced with rift locator mode, and the item locator mode should show all your stuff and MAYBE one of the things you're looking for.

Also hot damn the find objectives are ass. I had to find the disk, the captain's laser (which is just called 'laser gun' for crying out loud), the lawbringer, the hypospray and a rubber duck. Basically impossible without all-access or extreme violence.
